spampd.pl

```diff
@@ -392,6 +392,7 @@ BEGIN
 use Getopt::Long qw(GetOptions);
 use Time::HiRes qw(time);
 use Mail::SpamAssassin ();
+use Mail::SpamAssassin::Client ();
```

Comment:
Can we `use` or require this conditionally, if the client is going to be used?
For that matter maybe same with `Mail::SpamAssassin` also?

Reply:
Conditionally require client but in order to require `Mail::SpamAssassin` conditionally it will be a bigger share which can be done independently.